src/job_cache/daemon_cache.cpp

```diff
@@ -1140,8 +1139,18 @@ void DaemonCache::launch_evict_loop() {
 
     // Finally enter the eviction loop, if it exits cleanly
     // go ahead and exit with its result.
-    int result =
-        eviction_loop(".", std::make_unique<LRUEvictionPolicy>(max_cache_size, low_cache_size));
+    std::unique_ptr<EvictionPolicy> policy;
+    switch (config.type) {
+      case EvictionPolicyType::TTL:
+        wcl::log::info("Using TTL eviction policy")();
```

Comment:
I feel like theres no reason not to include the top-level details of these policies in the log (e.g. second to live for TTL and min and max size for LUR)